src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderEmploymentResponse.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@ApiModel(description = "Offender Employment")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class OffenderEmploymentResponse {
```

Comment:
I don't think we'd usually add `Response` to the object name, it's kind of superfluous.

Reply:
Is there another convention that is used? `Response` was to get around that the entity is called `OffenderEmployment`

Reply:
Yes, a good argument for rewriting this project in Kotlin and using import aliases :grin:  Maybe you could just call this `Employment`, the API is exposing Offender details and `Offender` has been omitted in most of the api model.